### PR TITLE
Xuan/fixing sharedbike type

### PR DESCRIPTION
**Pull request by Xuan-1998**
This change is [Reviewable](https://reviewable.io/reviews/lbnl-ucb-sti/beam/3544)

### CODE REVIEW

File: src/main/scala/beam/agentsim/agents/vehicles/BeamVehicleType.scala [hunk 1]
```diff
@@ -31,6 +31,7 @@ case class BeamVehicleType(
   chargingCapability: Option[ChargingPointType] = None,
   payloadCapacityInKg: Option[Double] = None
 ) {
+  def isSharedVehicle: Boolean = id.toString.startsWith("sharedVehicle")
```
Comment (dimaopen) [inline src/main/scala/beam/agentsim/agents/vehicles/BeamVehicleType.scala:34]: Method `beam.agentsim.agents.vehicles.BeamVehicle#isSharedVehicle` should call this method to have this code in one place.

**Review by dimaopen — CHANGES_REQUESTED** [review]
A single small change.

**Comment by Xuan-1998** [comment]
Test!

File: src/main/scala/beam/agentsim/agents/vehicles/BeamVehicleType.scala (hunk 1 above)
Comment (Xuan-1998) [reply src/main/scala/beam/agentsim/agents/vehicles/BeamVehicleType.scala:34]: adjusted, let me know if I understood it wrongly

**Comment by Xuan-1998** [comment]
Test!

**Comment by Xuan-1998** [comment]
Test!

**Review by dimaopen — APPROVED** [review]

**Pull request merged**